keps/sig-windows/2258-node-service-log-viewer/README.md

```diff
@@ -237,7 +247,7 @@ Options:
       --since='': Return logs after a specific ISO timestamp or relative date. Only applies to node journal or Get-WinEvent logs.
```

Comment:
I know that @thockin pointed that out, but this will be confusing for `kubectl logs` users, we have:
```
    --since=0s:
	Only return logs newer than a relative duration like 5s, 2m, or 3h. Defaults to all logs. Only one of
	since-time / since may be used.

    --since-time='':
	Only return logs after a specific date (RFC3339). Defaults to all logs. Only one of since-time / since may be
	used.
```
`--since` is to specify the duration, `--since-time` to specify the initial date, if you're not planning to have both, I'd ensure they align in naming and behavior.

Reply:
Consistency is more important that brevity, though `logs` has both `--since` and `--since-time` ?

Reply:
@thockin Should I add --since in addition? I was trying to keep the API as succinct as possible based on your previous feedback.

Reply:
Only if you KNOW you need it.  We can always add later